doc/conf.py

```diff
@@ -212,6 +220,18 @@ def setup(app):
     app.add_stylesheet('css/pygments.css')
     app.add_javascript("js/copybutton.js")
     app.add_javascript("js/custom.js")
+    app.add_config_value('recommonmark_config', {
+        'auto_toc_tree_section': 'Contents',
+        'enable_inline_math': True,
+        'enable_auto_doc_ref': True,
+        'enable_eval_rst': True
+    }, True)
+    app.add_transform(AutoStructify)
```

Comment:
I'm not 100% sure if we need this or if it was coming in through the merge from master. Also, the two calls to `app.add_config_value()` and `app.add_transform()` might be unrelated and the answer might differ for them. Please doublecheck.

Reply:
@jougs Both ``app.add_config_value()`` and ``app.add_transform()`` come from master and have been introduced following the merge of #1355. 

I can see from the new ``conf.py`` file that lines 40-41 and 229-240 have been added with #1355: https://github.com/nest/nest-simulator/blob/master/doc/conf.py

Weren't all ``recommonmark`` references removed in the past (#1263)? Or is there a specific reason for this addition, @terhorstd?

Please let us know!

Reply:
I thought all dependencies were removed. I am not aware that we use any markdown anymore, since all docs should be RST, afaik.